### PR TITLE
Fixes warning in PullToRefreshView demo

### PR DESCRIPTION
**Pull request by bestander**
Added a new PropType for Color type verification

### CODE REVIEW

**Comment by bestander** [comment]
cc @nicklockwood @ide

File: Libraries/PullToRefresh/PullToRefreshViewAndroid.android.js [hunk 1]
```diff
@@ -16,10 +16,12 @@ var View = require('View');
 
 var onlyChild = require('onlyChild');
 var processColor = require('processColor');
+var ColorPropType = require('ColorPropType');
```
Comment (ide) [inline Libraries/PullToRefresh/PullToRefreshViewAndroid.android.js]: This require should go above this group of lowercased requires

**Comment by nicklockwood** [comment]
It looks like the logic in the ColorPropType function is inverted, unless I've misunderstood how it works.

**Comment by nicklockwood** [comment]
Wouldn't it be better to define ColorPropType using the normal proptype composition functions? E.g.

    React.PropTypes.oneOfType([React.PropTypes.string, React.PropTypes.number]);

**Comment by bestander** [comment]
> It looks like the logic in the ColorPropType function is inverted, unless I've misunderstood how it works.
 @nicklockwood

It actually works. `React.PropTypes.*` return an Error object if check fails.
The ColorPropType outputs error only if both `number` and `string` check fail. 
But I think you are right, I just got lazy there.

> Wouldn't it be better to define ColorPropType using the normal proptype composition functions? 
That is actually a good idea.
Updating PR.

File: Libraries/StyleSheet/ColorPropType.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ *
+ * @providesModule ColorPropType
+ * @flow
+ */
+'use strict';
+
+var React = require('React');
+
+module.exports = function (props, propName, component) {
+  return React.PropTypes.oneOfType([React.PropTypes.string, React.PropTypes.number])(props, propName, component);
```
Comment (nicklockwood) [inline Libraries/StyleSheet/ColorPropType.js]: Maybe just

    module.exports = React.PropTypes.oneOfType([React.PropTypes.string, React.PropTypes.number]);

Comment (bestander) [inline Libraries/StyleSheet/ColorPropType.js]: Yep

**Comment by bestander** [comment]
What do you think of https://github.com/facebook/react-native/pull/4866?
Quite related

**Comment by nicklockwood** [comment]
There are a bunch of other places we use color props. Does it maybe make sense to update them all with the new `ColorPropType` in the same PR?

**Comment by nicklockwood** [comment]
> What do you think of #4866?
> Quite related

Yeah, I think his PR is better. Sorry ¯\_(ツ)_/¯

**Comment by bestander** [comment]
agree

**Pull request closed without merging**